### PR TITLE
Update mailing list sign up information

### PR DESCRIPTION
**Pull request by HelenCEBM**

### CODE REVIEW

File: frontend/templates/get_involved.html [hunk 1]
```diff
@@ -67,8 +67,8 @@ <h2>How can I get involved?</h2>
...
-      <li>Hopefully you are already making use of our <a href="https://openprescribing.net/">OpenPrescribing</a> service, including our email alerts service customised for your practice. We send a monthly OpenPrescribing newsletter in which we will mention any major OpenPathology news. You can sign up for this on the <a href="https://openprescribing.net">/OpenPrescribing.net home page</a>.</li>
+      <li>Stay in touch! Keep an eye on this website for updates, <a href="mailto:ebmdatalab@phc.ox.ac.uk">get in touch</a> with your thoughts/ideas or <a href="eepurl.com/gA3peX">sign up</a> to our OpenPathology mailing list to which we will send occasional updates and ask for input/feedback.</li>
+      <li>Hopefully you are already making use of our <a href="https://openprescribing.net/">OpenPrescribing</a> service, including our email alerts service customised for your <a href="https://openprescribing.net/practice">practice</a>.</li>
```
Comment (sebbacon) [inline frontend/templates/get_involved.html]: add "or CCG"?

Comment (HelenCEBM) [reply frontend/templates/get_involved.html]: this is the page "for GPs" so I just kept it as it was but we can add that if you like? No strong feeling either way

Comment (sebbacon) [reply frontend/templates/get_involved.html]: I think people other than GPs will end up there, and only linking to practice pages means we might miss out on a couple more signups (agree it's not super important but that's my feeling..)

Comment (HelenCEBM) [reply frontend/templates/get_involved.html]: I've added the CCG option now.

**Pull request merged**